### ISSUE TITLE
Organize sequential information into lists

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#585]
Issue group: Templates
Visibility: 2
Evaluation: Partially Supports
Success Criteria: 1.3.1: Info and Relationships
Needs design: Interaction

### Description
The previous messages are presented as siblings with headers containing the template name. However, because there are previous versions of the same template they, may all have the same name. It would be more appropriate to display these as an ordered list of versions. The currently applied version, should be marked with aria-current=“true”. The template name can be included in the page header.

![](https://dl.airtable.com/.attachments/c0e0f1b080045812c4389cde4fb436e9/4ec8f007/Capturedecranle2022-03-15a09_55_17.png)

**Comment by YedidaZalik** [comment]
@amazingphilippe worked on yesterday and running tests and will create PR

**Comment by adriannelee** [comment]
@amazingphilippe To take a look at comments

**Comment by amazingphilippe** [comment]
Merged and waiting for release, QA

**Comment by andrewleith** [comment]
Works and behaves as described.  Question though, @amazingphilippe, why can't we use `list-style: none;` to hide the markers? I think either that or we should stylize them so they look more intentional.

**Comment by amazingphilippe** [comment]
`list-style: none` will remove the list functionality with some assistive tech! https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html

**Comment by andrewleith** [comment]
🤦 this is why we can't have nice things

**Comment by andrewleith** [comment]
Just noticed this:
> This was a purposeful change due to rampant “list”-itis by web developers. … Basically, if you remove all default visible indication of the list, there is no indication to a sighted user or screen reader user that the content is a list. If you want to override this heuristic for accessibility, you can always add an explicit ARIA role=”list”.

So it might be possible to set `list-style: none` and then continue to have safari consider it as a list if we explicitly add `role="list"`.  I'll try to test this out when I have a few minutes! :)

**Comment by amazingphilippe** [comment]
That works! Although some a11y checkers will detect this as a redundant attribute. I think we use this technique in the account navigation maybe...
